Source/WebCore/platform/graphics/FloatRoundedRect.cpp

```diff
@@ -266,7 +266,7 @@ Region approximateAsRegion(const FloatRoundedRect& roundedRect, unsigned stepLen
         constexpr auto maximumCount = 20u;
         count = std::min(maximumCount, count);
 
-        for (auto i = 0u; i < count; ++i) {
+        for (size_t i = 0; i < count; ++i) {
```

Comment:
I believe `count` is an `int`, so there’s no good reason to use `size_t` here. If we change at all, I think we should consider just changing to match `count`.
```suggestion
        for (decltype(count) i = 0; i < count; ++i) {
```

Reply:
Resorted to `decltype(count)`